### PR TITLE
Issue 8 editing profiles

### PR DESCRIPTION
**Pull request by jechepenuk**
testing issues 38 & 39, starting from login page and from search page.

### CODE REVIEW

**Review by sedrehman — APPROVED** [review]
I check that once you log in with credentials, you can edit tutor's account and once you edited them you can see the change. then if you logged out and logged back in, you can see what you had changed is still there thus is working. GOOD JOB!!

File: style.css [hunk 1]
```diff
@@ -130,12 +129,22 @@ input[type="submit"]:hover {
...
-    text-align: center;
-    margin-top: 40px;
+    width: 50%;
+    margin-left: 25%;    
+    margin-right: 25%; 
+    text-align: center;   
+    margin-top: 140px;
 }
 .center{
     margin-top: 2%;
```
Comment (kesamercy) [inline style.css:150]: nit-picky but please indent your files correctly. if everyone pushes misaligned files it's going to be a hot mess.

**Review by kesamercy — CHANGES_REQUESTED** [review]
please rename your pages. you mix up the naming conventions by using camel case, or hyphens or sometimes just plain words and that's just confusing. Also, page names like update-process.. what process is it updating? what does that mean? could you pick an alternative better name? over-all look at revising your page names.

File: style.css (hunk 1 above)
Comment (jechepenuk) [reply style.css:150]: what files are indented wrong?

**Review by TKTre — APPROVED** [review]
Looking good!  Login works and tutor info functionality is well done.

**Review by kesamercy — APPROVED** [review]

**Pull request closed without merging**